website/docs/cli/cloud/settings.mdx

```diff
@@ -84,14 +84,16 @@ The `cloud` block supports the following configuration arguments:
 
 ### Environment Variables
 
-Alternatively, the `cloud` block can be configured with the following environment variables:
+Alternatively, you can configure the `cloud` backend non-interactively using environment variables, such as when configuring Terraform in a CI pipeline. The `cloud` block can be configured with the following environment variables:
+
+-> **Note:** These variables are only read if the corresponding attribute is omitted from your configuration. They behave as fallbacks rather than overriding your configuration. Furthermore, while all attributes can be omitted, an empty `cloud` block must still be present in your configuration.
```

Comment:
i think we can just make this a part of the introductory context (refer to my edit above)
```suggestion
```